sql/driver/conn.go

```diff
@@ -178,7 +179,8 @@ func (c *conn) Send(call sqlwire.Call) (*rows, error) {
 			} else if datum.IntVal != nil {
 				t[j] = *datum.IntVal
 			} else if datum.UintVal != nil {
-				t[j] = *datum.UintVal
+				// uint64 not supported by the driver.
```

Comment:
That comment is confusing.

```
uint64 is not supported by the driver.Value interface.
```

Also add a TODO to check for overflow in the conversion.